pkg/sentry/fsimpl/fuse/dev.go

```diff
@@ -168,6 +168,9 @@ func (fd *DeviceFD) readLocked(ctx context.Context, dst usermem.IOSequence, opts
 
 			// We're done with this request.
 			fd.queue.Remove(req)
+			if req.hdr.Opcode == linux.FUSE_RELEASE {
```

Comment:
`numActiveRequests` should include the requests that expect a response but not have one yet?

FUSE_RELEASE has a response.

duplicated decrement will potentially underflow it, and the `unique` is still in the completion map as well

Reply:
I don't think error response come from FUSE server, if you look at fuse_release_common -> fuse_file_put -> fuse_request_send_background in Linux, it only check if is still connected and set error accordingly (https://elixir.bootlin.com/linux/v4.4/source/fs/fuse/dev.c#L613).

Reply:
https://github.com/libfuse/libfuse/blob/50cbca799c1f9b010fabaa0dd4a387f29d140873/lib/fuse_lowlevel.c#L1457

Reply:
But if you want to decrement the `numActiveRequests`, better remove the request from the map as well. Otherwise it will underflow.

Reply:
It's a very weird protocol design: https://github.com/libfuse/libfuse/blob/7b3e3899157566875280a8b860eb5ad5c73eadc1/include/fuse_lowlevel.h#L618

Guess we just need to live with it.

Reply:
So there is no response from the server right for a release request? We should just remove it from the completions map and decrement the count then.